dynamic-configuration/src/k8s_questions.yaml

```diff
@@ -267,6 +267,53 @@ questions:
               func: returnasis
               args: ["false"]
 
+  - id: ssl_secret_options
+    question: "How do you want to configure SSL certification?:\n1. Ingress record annotation to issue the certificate by using an issuer (issue must be installed in the cluster)\n2. Other\n:"
+    var: SSL_SECRET_OPTIONS
+    default: ""
+    validation:
+      - func: onlythesevalues
+        args: ["$SSL_SECRET_OPTIONS", "1,2"]
+        success: noerror
+        err_msg: "Invalid input. Please enter '1' or '2'."
+    post_processing: []
+    next:
+      value: "$SSL_SECRET_OPTIONS"
+      conditions:
+        - when: "1"
+          question_id: ingress_annotation
+          action: proceed
+          expose: []
+        - when: "2"
+          question_id: own_ssl_secret
+          action: proceed
+          expose: []
+
+  - id: ingress_annotation
+    question: "Please provide the annotation key and label: (eg. cert-manager.io/cluster-issuer=\"letsencrypt-production\")"
```

Comment:
The example already shows it, but maybe we can add a short sentence indicating that escape characters need to be used for double quotes

Reply:
user shouldn't see the escapes, no?

Reply:
Oh the escape is just for the question itself? In that case nevermind haha